web/client/components/widgets/builder/wizard/map/enhancers/__tests__/withSelectedNode-test.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018, GeoSolutions Sas.
```

Comment:
Hi, for all new files, we use the current year as copyright year value

```suggestion
 * Copyright 2020, GeoSolutions Sas.
```

Reply:
Updated